companies/lush.md

```diff
@@ -0,0 +1,35 @@
+---
+
+name: Lush
+tags: ["retail"]
+image: lush.png
+blm_statements:
+  - url: https://twitter.com/lushcosmetics/status/1266485725400150016
+    date_posted: 2020-05-29
+    type: Tweet
+    raw_text: >
+      We say enough.
+ 
```

Comment:
```suggestion

```

@cody-castro do you mind cleaning up these 'trailing spaces' ? I think the alerts are just for empty lines that have a blank space on them.